### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by pasd0447**
Basic Dashboard

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -6,8 +6,8 @@ APP_URL=http://localhost
 
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
-DB_PORT=3306
-DB_DATABASE=homestead
+DB_PORT=33060
```
Comment (itsmattli) [inline .env.example]: we should leave this unchanged as its just the template

File: app/Http/Kernel.php [hunk 2]
```diff
@@ -30,7 +30,7 @@ class Kernel extends HttpKernel
             \App\Http\Middleware\EncryptCookies::class,
             \Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse::class,
             \Illuminate\Session\Middleware\StartSession::class,
-            // \Illuminate\Session\Middleware\AuthenticateSession::class,
+            \Illuminate\Session\Middleware\AuthenticateSession::class,
```
Comment (itsmattli) [inline app/Http/Kernel.php]: i think this is unnecessary

File: config/database.php [hunk 3]
```diff
@@ -1,108 +0,0 @@
-<?php
```
Comment (itsmattli) [inline config/database.php]: we should discard this changes as we shouldn't be changing the default database.php file

File: resources/views/layouts/app.blade.php [hunk 4]
```diff
@@ -12,9 +12,14 @@
 
     <!-- Styles -->
     <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+    <link href="{{ asset('css/sidebar.css') }}" rel="stylesheet" />
+
 
-    <link rel="stylesheet" href="{{ asset('css/bootstrap/bootstrap-theme.min.css')}}">
```
Comment (itsmattli) [inline resources/views/layouts/app.blade.php]: @brynbeaudry I'm not too sure about this, but I don't think we should be removing bootstrap from the default layout file.

File: public/css/bootstrap/bootstrap.min.css [hunk 5]
```diff
@@ -0,0 +1,5 @@
+/*!
```
Comment (nikoootine) [inline public/css/bootstrap/bootstrap.min.css]: we dont need to import bootsrap again as its added to laravel by default. see app.css

File: resources/views/layouts/app.blade.php (hunk 4 above)
Comment (nikoootine) [inline resources/views/layouts/app.blade.php]: did you remove this because of the error in the browser? dont remove this, import the glyphicons instead inside the css folder

File: config/database.php (hunk 3 above)
Comment (nikoootine) [inline config/database.php]: this file was deleted from the project folder instead of discarding the changes.

please make another commit to this branch that includes this file
